Buildings/Fluid/HeatExchangers/AirToAirHeatRecovery/WheelWithVariableSpeed.mo

```diff
@@ -0,0 +1,227 @@
+    annotation (Line(points={{-19,54},{-6,54},{-6,4},{4,4}}, color={0,0,127}));
+  connect(effCal.epsLat, hex.epsLat)
+    annotation (Line(points={{-19,46},{-12,46},
+    {-12,-4},{4,-4}}, color={0,0,127}));
+  connect(effCal.wheSpe, wheSpe)
+    annotation (Line(points={{-42,50},{-94,50},{-94,
+    0},{-120,0}}, color={0,0,127}));
+  connect(senSupFlow.V_flow, effCal.VSup_flow)
+    annotation (Line(points={{-35,108.8},
+    {-35,120},{-80,120},{-80,58},{-42,58}}, color={0,0,127}));
+  connect(senExhFlow.V_flow, effCal.VExh_flow)
+    annotation (Line(points={{49,2.8},
+    {49,80},{-74,80},{-74,54},{-42,54}}, color={0,0,127}));
+  connect(TSup.y, effCal.TSup)
+    annotation (Line(points={{-69,38},{-60,38},{-60,46},
+    {-42,46}}, color={0,0,127}));
+  connect(TExh.y, effCal.TExh)
+    annotation (Line(points={{-69,20},{-52,20},{-52,42},
+    {-42,42}}, color={0,0,127}));
+  annotation (
+        defaultComponentName="whe",
+        Icon(coordinateSystem(extent={{-100,-100},{100,
+            100}}), graphics={
+        Rectangle(
+          extent={{32,-56},{94,-64}},
+          lineColor={0,0,255},
+          pattern=LinePattern.None,
+          fillColor={0,0,0},
+          fillPattern=FillPattern.Solid),
+        Rectangle(
+          extent={{32,64},{94,56}},
+          lineColor={0,0,255},
+          pattern=LinePattern.None,
+          fillColor={0,0,0},
+          fillPattern=FillPattern.Solid),
+        Rectangle(
+          extent={{-92,-55},{-32,-64}},
+          lineColor={0,0,255},
+          pattern=LinePattern.None,
+          fillColor={0,0,0},
+          fillPattern=FillPattern.Solid),
+        Rectangle(
+          extent={{-94,65},{-32,56}},
+          lineColor={0,0,255},
+          pattern=LinePattern.None,
+          fillColor={0,0,0},
+          fillPattern=FillPattern.Solid),
+        Ellipse(extent={{6,88},{38,-90}},   lineColor={28,108,200},
+          fillColor={255,255,255},
+          fillPattern=FillPattern.Solid),
+        Polygon(
+          points={{0,100},{0,100}},
+          lineColor={28,108,200},
+          fillColor={255,255,255},
+          fillPattern=FillPattern.None),
+        Rectangle(
+          extent={{-2,88},{22,-98}},
+          fillColor={255,255,255},
+          fillPattern=FillPattern.Solid,
+          pattern=LinePattern.None),
+        Ellipse(
+          extent={{-38,88},{-4,-90}},
+          lineColor={28,108,200},
+          fillColor={28,108,200},
+          fillPattern=FillPattern.Solid),
+        Line(points={{-22,-90},{22,-90}},   color={28,108,200}),
+        Line(points={{-20,88},{22,88}},   color={28,108,200}),
+        Text(
+          extent={{-149,-108},{151,-148}},
+          textColor={0,0,255},
+          textString="%name")}),
+          Diagram(
+        coordinateSystem(preserveAspectRatio=false, extent={{-100,-100},{100,140}})),
+Documentation(info="<html>
+<p>
+Model of a generic, sensible, and latent air-to-air heat recovery wheel with a variable speed drive.
```

Comment:
`Model of a generic, sensible and latent air-to-air heat recovery wheel, which has the wheel speed as the input to control the heat recovery.`